package.json

```diff
@@ -36,17 +36,18 @@
     "postpack": "ember ts:clean"
   },
   "dependencies": {
-    "@types/sinon": "^10.0.13",
+    "@types/sinon": "^10.0.19",
     "broccoli-funnel": "^3.0.8",
-    "ember-auto-import": "^2.6.1",
-    "ember-cli-babel": "^7.26.11",
+    "ember-auto-import": "^2.6.3",
+    "ember-cli-babel": "^8.2.0",
```

Comment:
this is not a req for ember v5, right?

Reply:
I was having an issue with Babel [babel/preset-env](https://babeljs.io/docs/babel-preset-env) in some tests.

Reply:
Ok, I rollbacked this one.